custom/_legacy/pact/templates/pact/patient/pactpatient_careplan.html

```diff
@@ -1,6 +1,9 @@
 {% extends "pact/patient/pactpatient_base.html" %}
 {% load case_tags %}
+{% load hq_shared_tags %}
+
 {% block patient-tab-container %}
+<script src="{% new_static 'jquery-timeago/jquery.timeago.js' %}"></script>
```

Comment:
@dimagi/js-team it looks like the timeago import used to be in bootstrap2/base.html but it's not in boostrap3/base.html anymore. should it be, or should we just import it here directly? i think this issue was introduced when this report was migrated to bootstrap 3, and this report will face the same problem when it's upgraded: https://github.com/dimagi/commcare-hq/blob/pact-fix/corehq/apps/reports/templates/reports/careplan/patient_careplan.html